src/routeGeneration/routeGenerator.ts

```diff
@@ -13,26 +13,33 @@ export class RouteGenerator {
     const fileName = `${this.options.routesDir}/routes.ts`;
     const content = this.buildContent(middlewareTemplate, pathTransformer);
 
-    return new Promise<void>((resolve, reject) => {
-      tsfmt.processString(fileName, content, {
-        editorconfig: true,
-        replace: true,
-        tsconfig: true,
-        tsfmt: true,
-        tslint: true,
-        verify: true,
-        vscode: true
-      } as any)
-        .then(result => {
-          fs.writeFile(fileName, result.dest, (err) => {
+    return new Promise<any>((resolve, reject) => {
+      fs.writeFile(fileName, content, (err) => {
             if (err) {
               reject(err);
             } else {
               resolve();
             }
           });
-        }
-        );
+      // tsfmt.processString(fileName, content, {
```

Comment:
fixed. It turned out that the error happened because of invalid config of my IDE (vscode).